proto/confio/v1/proofs.proto

```diff
@@ -2,15 +2,17 @@ syntax = "proto3";
 
 package ics23;
```

Comment:
I would propose changing this to condo.v1.ics23 to have version proto files. thoughts?